MagneticField/ParametrizedEngine/interface/ParabolicParametrizedMagneticField.h

```diff
@@ -20,17 +25,32 @@ namespace magneticFieldParabolicPortable {
 
   template <typename Vec3>
   constexpr float Kr(Vec3 const& vec) {
-    return Parameters::a * (vec(0) * vec(0) + vec(1) * vec(1)) + 1.f;
+    return Parameters::a * (vec[0] * vec[0] + vec[1] * vec[1]) + 1.f;
```

Comment:
I guess the `operator[]` works also with `Eigen::Matrix<float, 3, 1>` (that the unit test uses). Arguably for 1D vector the `operator[]` is more general as works e.g. with C-array and `std::array`.